data/projects/implement_aspects.md

```diff
@@ -7,9 +15,26 @@ The aspects concept exists as
 already possible to define aspects and use them in bears - however this has no
 effect because coala doesn't understand what to do with them yet.
 
```

Comment:
Perhaps you should give a brief overview of aspects right here? Something that describes how this will be implemented briefly?